sdk/search/ci.yml

```diff
@@ -3,9 +3,9 @@
 
 resources:
   repositories:
-    # - repository: azure-sdk-build-tools
-    #   type: git
-    #   name: internal/azure-sdk-build-tools
+     - repository: azure-sdk-build-tools
+       type: git
+       name: internal/azure-sdk-build-tools
```

Comment:
`yml` files are space sensitive, this is using the wrong character offset. This will either lead to a parsing exception or the configuration being ignored.